libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/MapboxSpeechPlayer.java

```diff
@@ -113,7 +123,7 @@ private void flushCache() {
     try {
       okhttpCache.flush();
     } catch (IOException exception) {
-      Timber.e(exception.getMessage());
```

Comment:
shouldn't this stay how it is, printing the exception's message instead of toString?

Reply:
`Timber` was showing a lint warning here `Explicitly including the exception message is redundant when supplying an exception to the log`.  

It looks like the library includes the exception message as well as the rest of the exception data when you give it the `Exception` as a param.